README.md

```diff
@@ -38,9 +40,81 @@ Detailed version can be seen on [Codecov.io][coverage-link]
 https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggVERcbiAgdm17Vk19XG4gIGNvbW1vbiAtLT4gYmxvY2tjaGFpblxuICBjb21tb24gLS0-IGJsb2NrXG4gIGNvbW1vbiAtLT4gdm1cbiAgY29tbW9uIC0tPiB0eFxuICBldGhhc2ggLS0-IGJsb2NrY2hhaW5cbiAgYmxvY2sgLS0-IGJsb2NrY2hhaW5cbiAgYmxvY2tjaGFpbiAtLT4gdm1cbiAgYmxvY2sgLS0-IHZtXG4gIHR4IC0tPiB2bVxuICB0eCAtLT4gYmxvY2tcbiAgYWNjb3VudCAtLT4gdm1cbiAgIiwibWVybWFpZCI6eyJ0aGVtZSI6ImRlZmF1bHQifSwidXBkYXRlRWRpdG9yIjpmYWxzZX0
 -->
 
-## Developing in a monorepo
+## Development quick start
+
+This monorepo uses [Lerna](https://lerna.js.org/). It links the local packages together, making development a lot easier.
+
+TLDR: Setup
+```sh
+npm install
+npm build
+```
+
+TLDR: To update dependencies and (re-)link packages
+```sh
+npm run bootstrap
+npm build
+```
+
+Above is the quickest way to set you up. Going down the road, there are two sets of commands: *project* and *package-specific* commands. You can find them at `./package.json` and `./packages/*/package.json`, respectively. Here's a breakdown:
+
+### Project scripts — run from repository root
+
+#### `npm install`
+Adds dependencies listed in the root package. Also, it executes the `bootstrap` script described below, installing all sub-packages dependencies.
+
+#### `npm run bootstrap`
+
+Installs dependencies for all sub-packages, and links them to create an integrated development environment.
+
+#### `npm run build`
+
+Produces `dist` files for all sub-packages. This command can be scoped
+
+#### `npm run build:tree -- --scope @ethereumjs/blockchain`
+
+Builds all local packages that the provided package (eg: @ethereumjs/blockchain) depends on, and itself. This unusual syntax just means: pass whatever arguments are after `--` to the underlying script. 
+
+If no scope is provided, `npm run build:tree`, will build all sub-packages.
+
+### Package scripts — run from `./packages/<name>`
+
+ **⚠️ Important: if you run `npm install` from the package directory, it will remove all links to the local packages, pulling all dependencies from npm.**
```

Comment:
Oh, that's a nice hint and being **bold** very justified, I would likely never have gotten this myself and randomly run into for quite some time.